docs/eventing/sources/apiserversource/getting-started/README.md

```diff
@@ -1,32 +1,36 @@
----
-title: "Getting started"
-weight: 01
-type: "docs"
-showlandingtoc: "false"
-aliases:
-  - /docs/eventing/samples/kubernetes-event-source
----
+# Creating an ApiServerSource object
 
-# Getting started
+![version](https://img.shields.io/badge/API_Version-v1-red?style=flat-square)
 
-## Prerequisites
+This topic describes how to create an ApiServerSource object.
 
-Before you can create an API server source, you must install Knative Eventing and the `kubectl` CLI tool.
+## Before you begin
 
-## Create an API server source
+Before you can create an ApiServerSource object:
 
-1. Optional: Create a namespace for the API server source instance:
+- You must have [Knative Eventing](../../../admin/install/install-eventing-with-yaml)
```

Comment:
This link doesn't work in Netlify preview: https://deploy-preview-3812--dev-knative.netlify.app/development/eventing/admin/install/install-eventing-with-yaml

Not sure if it is an actual problem